crates/test-environment/src/services.rs

```diff
@@ -37,6 +37,13 @@ impl Services {
                     Box::new(DockerService::start(&service_def.name, image, &lock_dir)?)
                 }
             };
+            // TODO: Postgres seems to show Ready before the DB is actually ready to accept connections.
```

Comment:
I am not sure what to do about this - the PostgreSQL container becomes "healthy" quite early in its load cycle, but still takes a few seconds before it starts accepting connections.  So without this the tests were failing with an "early eof".  I guess this would be better handled by a more custom health test or something?  Or a new service/test setting to specify a wait time?  Open to recommendations.